src/views/Point.js

```diff
@@ -44,7 +46,7 @@ export default function Point() {
       <Passport point={Maybe.Just(point)} />
       <Grid className="pt2">
         <Grid.Item full>
-          <ForwardButton disabled>Admin</ForwardButton>
+          <ForwardButton onClick={goAdmin}>Admin</ForwardButton>
```

Comment:
only allow going to the Admin page if the pointCache has been set, otherwise the Admin page will throw because of the `need` requirements.

Reply:
Oh, that commit's sitting unpushed, haha.